src/views/webviewController.ts

```diff
@@ -100,7 +100,7 @@ export default class WebviewController {
               action: message.action,
               files:
                 files && files.length > 0
-                  ? files.map((file) => path.resolve(file.path))
+                  ? files.map((file) => path.resolve(file.path.substr(1)))
```

Comment:
Windows was building the file string as `C:/C:/somepath/to/file.txt` I think this should cover us because vscode is giving us absolute file paths. Tested on windows and now we're getting the right file path. Also tested SSL on linux with a file path and it connected with shell.